routes/login-routes.js

```diff
@@ -0,0 +1,37 @@
+const db = require("../models");
+
+module.exports = function(app){
+
+
+    // POST new user
+    app.post("/api/users", function(req, res){
+        db.Products.create(req.body).then(function(dbUser){
+            res.json(dbUser)
```

Comment:
Prefer arrow functions for short functions without this.
.then((dbUser) => res.json(dbUser)

Reply:
Thank you, I'll go through and switch to ES6 arrow functions where appropriate.